public/main.css

```diff
@@ -0,0 +1,84 @@
+.bg-1 {
+  background-color: rgb(74, 115, 129);
+  color: #ffffff;
+}
+.bg-2 {
+background-color: rgb(74, 115, 129);
+color: #ffffff;
+}
+.bg-3 {
+  background-color: white;
+  color: #555555;
+}
+
+.address p {
+  line-height: .5;
+}
+
+.bg-3 img {
+  width: 3%;
+  height: 3%;
+  margin-top: 20px
+}
+
+.bg-0 {
+  background-color: rgb(108, 152, 167);
+  color: rgb(32, 28, 28);
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  flex: 50%;
+  padding: 5px;
+}
+
+.bg-1 h2 {
+  background-color: rgb(108, 152, 167);
+  color: rgb(32, 28, 28);
+  text-align: center;
+}
+
+h1 {
+  color: rgb(32, 28, 28);
+}
+
+h3 {
+  color: rgb(32, 28, 28);
+  font-size: 15px;
+  line-height: 1.5;
+  margin-left: 80px;
+  margin-right: 80px;
+  text-align: center;
+}
+
+img {
```

Comment:
You probably don't want to make _every_ `<img>` tag on the site a block element. If you want to make every `<img>` inside a carousel item, do something like...

```css
.carousel .item img {
  display: block;
}
```